docs/connect/jdbc/understanding-data-type-conversions.md

```diff
@@ -106,7 +106,7 @@ There are three categories of conversions that are supported by the JDBC driver'
 
 - **Data Dependent (z)**: Conversions from a Java **String** type to the underlying [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] data type depends on the following conditions: The driver sends the **String** value to [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] and [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] performs conversions, if necessary. If the sendStringParametersAsUnicode connection property is set to true and the underlying [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] data type is **image**, [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] doesn't allow converting **nvarchar** to **image** and throws an SQLServerException. If the sendStringParametersAsUnicode is set to false and the underlying [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] data type is **image**, [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] allows converting **varchar** to **image** and doesn't throw an exception.
 
-[!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] performs the bulk of the set conversions and passes errors back to the JDBC driver when there are problems. Client-side conversions are the exception and are performed only in the case of **date**, **time**, **timestamp**, **Boolean**, and **String** values.
+[!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] performs the bulk of the set conversions and passes errors back to the JDBC driver when there are problems. Client-side conversions are the exception and are performed only in the case of **date**, **time**, **rowversion**, **Boolean**, and **String** values.

```

Comment:
This should be reverted.

In this context, the doc is referring to the JDBC timestamp data type, not the SQL Server timestamp data type. If you look at the [graphic on the page](https://docs.microsoft.com/en-us/sql/connect/jdbc/understanding-data-type-conversions?view=sql-server-ver15#conversions-on-setobject), you can see that the only references are to SQL Server datetime, smalldatetime, date, time, datetime2, and datetimeoffset types.